handler.go

```diff
@@ -2057,3 +2084,41 @@ func (h *Handler) handleGetID(w http.ResponseWriter, r *http.Request) {
 }
 
 type defaultClusterMessageResponse struct{}
+
+type queryValidationSpec struct {
+	required []string
+	args     map[string]bool
```

Comment:
If you aren't going to use the `bool` value for anything, this could be `map[string]struct{}`.

Reply:
ok